src/extensions/codemirror/_codemirror.scss

```diff
@@ -22,6 +24,73 @@ $eccgui-color-codeeditor-background: $eccgui-color-textfield-background !default
         // get them a "border" like input boxes from blueprintjs
         box-shadow: input-transition-shadow($input-shadow-color-focus), $pt-input-box-shadow;
 
+        &.#{eccgui}-disabled {
+            @extend .#{$ns}-input, .#{$ns}-disabled;
+
+            min-height: 290px;
```

Comment:
This (and the other) `min-height: 290px;` rule leads to a visual bug here: https://62150dbccb2d77003a2a5093-dulpfvcgaz.chromatic.com/?path=/story/forms-codeautocompletefield--default